dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/storage/ImageTagStore.java

```diff
@@ -0,0 +1,26 @@
+package com.salesforce.dockerfileimageupdate.storage;
+
+import com.salesforce.dockerfileimageupdate.utils.DockerfileGitHubUtil;
+
+import java.io.IOException;
+import java.util.Map;
+
+public interface ImageTagStore {
+
+    /**
+     * This method updates the image tag store by updating the image version for the image name passed.
+     *
+     * @param img the name of the image that needs to be updated.
+     * @param tag the version of the image that it needs to update to.
+     */
+    void updateStore(String img, String tag) throws IOException;
+
+    /**
+     * This method gets the content of the image tag store.
+     *
+     * @param dockerfileGitHubUtil the dockerfileGitHubUtil object that is used to interact with an underlying Git repo.
+     * @param storeName the name of the store whose content needs to be fetched.
+     * @return A Map of image name to image version.
+     */
+    Map<String, String> getStoreContent(DockerfileGitHubUtil dockerfileGitHubUtil, String storeName) throws IOException, InterruptedException;
```

Comment:
Instead of `Map<String, String>` just create a new class to store the data you want.